[API Shield] Volumetric Abuse Detection revision (#9602)
cloudflare/cloudflare-docs: content/api-shield/security/volumetric-abuse-detection.md

Cloudflare recommends choosing the overall rate limit recommendation, as our analysis includes the variance of the request rate distribution across your API sessions. Choosing a single p-value may cause false positives due to a high number of outliers.

{{<Aside type="Note">}}
The p-values describe what percentile of your traffic fits below the value. For example, if your p90 value is 83, then 90% of your sessions had maximum request rates less than 83 requests per 10 minutes.
{{</Aside>}}

You can also see our confidence in the recommendation and how many unique sessions we have seen over the last 7 days. In general, endpoints with fewer unique sessions and high variability of user behavior will have lower confidence scores.

{{<Aside type="Note">}}
Implementing low confidence rate limits can still be helpful to prevent API abuse. If you are hesitant due to the recommendation’s confidence, we suggest starting your rate limit rule in `log` mode and observing violations of the rule for false positives.
{{</Aside>}}

### Create rate limits

> In the endpoint row, select **Create rule** to be automatically redirected to the [Advanced Rate Limiting](/waf/rate-limiting-rules/create-zone-dashboard/) rules dashboard with your endpoint information, session identifier, and recommended rate limit pre-filled into the rule. Give your rule a name, action, and duration. Then, select **Deploy** to activate your rule.

Reviewer: This section should change to

To <accomplish task>:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com), and select your account.
2. Step 
3. Step
4. Step
5. Select **Deploy** to activate your rule.

## API

> Rate limit recommendations [are available via the API](/api/operations/api-shield-endpoint-management-retrieve-information-about-an-operation) if you would like to dynamically update rate limits over time.

Reviewer: ```suggestion
[Rate limit recommendations are available via the API](/api/operations/api-shield-endpoint-management-retrieve-information-about-an-operation) if you would like to dynamically update rate limits over time.
```
PCX > Style Guide > UX > Content accessibility > UI content accessibility best practices

> ## Limitations

Author: Other pages in the product docs call it Limitations! will keep as is for consistency

> While API Shield will always calculate recommendations when session identifiers are configured, enabling session-based rate limits requires a subscription to Advanced Rate Limiting.

Reviewer: ```suggestion
API Shield will always calculate recommendations when session identifiers are configured. To enable session-based rate limits, subscribe to Advanced Rate Limiting.
```
Instructions on how to subscribe to Advanced Rate Limiting needed.

These two scenarios speak to the limitations of traditional rate limiting. Not only does traffic vary between endpoints, but it also can vary over time for the same endpoint. Volumetric Abuse Detection solves these problems with unsupervised learning to develop separate baselines for each API and better adjust to changes in user behavior.

Volumetric Abuse Detection rate limits are generated on a per-session basis. Unlike traditional rate limits, which are based on IP addresses, Volumetric Abuse Detection rate limits are not as susceptible to false positives when traffic to your API increases.

Volumetric Abuse Detection rate limits are a way to prevent blatant volumetric abuse while minimizing false positives. If you are trying to prevent abusive bot traffic altogether, refer to Cloudflare’s [Bot solutions](/bots/).

## Process

Volumetric Abuse Detection analyzes your API’s individual session traffic statistics to recommend per-endpoint, per-session rate limits.

> This feature currently requires a [session identifier](/api-shield/get-started/#set-up-session-identifiers), like an authorization token available as a request header or cookie.

Reviewer: ```suggestion
Volumetric Abuse Detection currently requires a [session identifier](/api-shield/get-started/#set-up-session-identifiers), like an authorization token available as a request header or cookie.
```

> After adding a session identifier, allow 24 hours for rate limit recommendations to appear on endpoints in **Endpoint Management** in the Cloudflare dashboard. Recommendations will continue to update if your traffic pattern changes

Reviewer: ```suggestion
After adding a session identifier, allow 24 hours for rate limit recommendations to appear on endpoints in **Security** > **API Shield** > **Endpoint Management** on the Cloudflare dashboard. Recommendations will continue to update if your traffic pattern changes
```

Reviewer: felt weird to say 'in' twice so changed one to on.

Cloudflare recommends choosing the overall rate limit recommendation, as our analysis includes the variance of the request rate distribution across your API sessions. Choosing a single p-value may cause false positives due to a high number of outliers.

{{<Aside type="note" header="p-values">}}
p-values describe what percentile of your traffic fits below the value. For example, if your p90 value is `83`, then 90% of your sessions had maximum request rates less than 83 requests per 10 minutes.
{{</Aside>}}

In **Endpoint Management**, you can review our confidence in the recommendation and how many unique sessions we have seen over the last seven (7) days. In general, endpoints with fewer unique sessions and high variability of user behavior will have lower confidence scores.

{{<Aside type="Note">}}
Implementing low confidence rate limits can still be helpful to prevent API abuse. If you are hesitant due to the recommendation’s confidence, we suggest starting your rate limit rule in `log` mode and observing violations of the rule for false positives.
{{</Aside>}}

> ### Create rate limits
> 
> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.

Reviewer: ```suggestion
### Create rate limits

To create rate limits:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.
```
